app/models/reports/result.rb

```diff
@@ -173,9 +187,21 @@ def calculate_period_info(range)
       end
     end
 
-    def registrations_for_percentage_calculation(period)
+    def registrations_for_percentage_calculation_with_ltfu(period)
+      if quarterly_report?
+        self[:registrations_with_ltfu][period.previous] || 0
+      else
+        adjusted_registrations_with_ltfu_for(period)
+      end
+    end
+
+    def registrations_for_percentage_calculation(period, key)
+      if key == :controlled_patients_with_ltfu
+        return registrations_for_percentage_calculation_with_ltfu(period)
+      end
```

Comment:
Not super fond of this, but basically we want to be able to switch the denominator when calculating control rate for `with_ltfu` patients.